proc_load_old.py

```diff
@@ -0,0 +1,140 @@
+'''
+Load data in parallel with train.py
+'''
+
```

Comment:
Is this file necessary to keep? how about make doing the RGB augmentation as an option?